concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -2,6 +2,14 @@
 
 import com.concordium.sdk.exceptions.*;
 import com.concordium.sdk.responses.AccountIndex;
+import com.concordium.sdk.responses.branch.Branch;
```

Comment:
The exceptions are already imported via the * above. Same counts for 
```
import com.concordium.sdk.responses.blocksatheight.BlocksAtHeight;
import com.concordium.sdk.responses.peerStats.PeerStatistics;
import com.concordium.sdk.responses.peerlist.Peer;
```

Reply:
done